docs/docs/changelog/changelog.md

```diff
@@ -2,6 +2,10 @@
 
 ## [8.7.0](https://github.com/equinor/ecalc/compare/v8.6.0...v8.7.0) (2023-12-05)
 
+### ⚠ BREAKING CHANGES
+
+* Change name from `DIRECT_EMITTERS` to `VENTING_EMITTERS` in input Yaml-file ([#303](https://github.com/equinor/ecalc/pull/303))
```

Comment:
Is this file generated? Or why are we duplicating this in next?

Reply:
this file is generated yes, we should not alter "that history". It was included in 8.6.0 afaik, should not be stating in 8.7.0 changes then

Reply:
The changelog.md is now rolled back, no change.